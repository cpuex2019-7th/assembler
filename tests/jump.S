label1:
    addi t0, zero, 1
    j label3
label2:
    li t3, 0x12345678
    addi t1, t1, 2
    j ret
label3:
    addi t1, zero, 3
    j label2
ret:
    add a0, t0, t1
