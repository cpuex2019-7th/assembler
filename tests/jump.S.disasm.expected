addi t0, zero, 1
jal zero, 20
lui t3, 74565
addi t3, t3, 1656
addi t1, t1, 2
jal zero, 12
addi t1, zero, 3
jal zero, -20
add a0, t0, t1
